serokell-style.md

```diff
@@ -418,19 +413,19 @@ putValueInState
 
 In other cases place `=` sign on the same line where function definition is.
 
-Use `() <$` to  ignore the result of function. It looks cool. If this is not possible
-due to `$` then use either `_ <-` or `void $`.
+Do not use `() <$` to ignore the result of function, because it harms readability.
+Instead, use either `_ <-` or `void $`.
 
 ```haskell
 foo = do
-    _  <- forkIO $ myThread config  -- can't be used as last statement
-    () <$ sendTransactionAndReport 1 "tx42"
+  _  <- forkIO $ myThread config  -- cannot be used as last statement
```

Comment:
Ignoring the result of `forkIO` is usually a bad thing. I'd change this example (even though it's not particularly about `forkIO` of course).